lib/DDG/Fathead/Abbreviations.pm

```diff
@@ -2,7 +2,7 @@ package DDG::Fathead::Abbreviations;
 
 use DDG::Fathead;
 
-primary_example_queries "what does ASAP mean";
+primary_example_queries "ASAP abbrevation";
```

Comment:
@jdorweiler sorry, my bad, that was a typo on my side.

I updated both of them so that they are the same as in the IA pages.